packages/cli/cli/src/commands/upgrade/upgrade.ts

```diff
@@ -38,12 +38,15 @@ const PREVIOUS_VERSION_ENV_VAR = "FERN_PRE_UPGRADE_VERSION";
 export async function upgrade({
     cliContext,
     includePreReleases,
+    upgradeVersion,
 }: {
     cliContext: CliContext;
     includePreReleases: boolean;
+    upgradeVersion?: string;
```

Comment:
`upgradeVersion: string | undefined` (reasoning: if we forget to pass it, that's probably a mistake on our end, so we'd want the compiler to warn)

Reply:
also nitty but can we call it `toVersion` or `targetVersion` or something like that